Makefile

```diff
@@ -349,8 +352,36 @@ $(PROTOC):
 	unzip -oq ${PROTOC_ZIP} -d $(CACHE) 'include/*'; \
 	rm -f ${PROTOC_ZIP})
 
+grpc-gateway:
+	@echo "Installing protoc-gen-grpc-gateway..."
+	@rm -f $@
+	@curl -o "${GOBIN}/protoc-gen-grpc-gateway" -L \
+	"https://github.com/grpc-ecosystem/grpc-gateway/releases/download/v${PROTOC_GRPC_GATEWAY_VERSION}/${PROTOC_GRPC_GATEWAY_BIN}"
+	chmod +x "${GOBIN}/protoc-gen-grpc-gateway"
+
+protoc-swagger:
+ifeq (, $(shell which protoc-gen-swagger))
+	@echo "Installing protoc-gen-swagger..."
+	@go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
```

Comment:
Add this to `tools.go` and install along with everything else from there in `devdeps-install`.